rjc_account_form/reports/invoice_credit_note_layout.xml

```diff
@@ -0,0 +1,204 @@
+<odoo>
+    <data>
+        <template id="rjc_invoice_credit_note_layout_report">
+            <div class="article">
+                <div class="header">
+                    <div class="row">
+                        <div class="col-3">
+                            <span>
+                                <img src='/rjc_account_form/static/src/img/raja.jpg' style="max-height: 75px;"/>
+                            </span>
+                            <br>
+                                <strong>บริษัท ราชาไซเคิล จำกัด</strong>
+                                <strong>RAJA CYCLE CO.,LTD.</strong>
+
+                            </br>
+                        </div>
+                    </div>
+                    <div class="row">
+                        <div class="col-2" style="margin-left:350px">
```

Comment:
```suggestion
                        <div class="col-12">
```
why used col-2 and style?
number after col is mean blog in page
i.e.
Paper has 12 column
1|2|3|4|5|6|7|8|9|10|11|12

if you used `col-2`. it mean you used 2 column in page.
if text lenght more than 2 column, it's new line
her|3|4|5|6|7|8|9|10|11|12
e   |3|4|5|6|7|8|9|10|11|12

if you used `col` or `col-12`
`here` is full page